background/about-boilerplates.rst

```diff
@@ -0,0 +1,151 @@
+Boilerplates implement these two popular frontend frameworks.
+
+They are both fully-featured frameworks that include opinionated CSS and
+JavaScript for your own use, and numerous built-in widgets and standardised web
+components.
+
+
+Sass Boilerplate
+~~~~~~~~~~~~~~~~
+
+Our `Sass Boilerplate <https://github.com/divio/djangocms-boilerplate-sass>`_
+introduces compiled components. This is a *dynamic Boilerplate*, unlike those
+above, which is to say that it compiles its own materials at runtime (the
+*static Boilerplates* by contrast simply use or serve the materials they ship
+with).
+
+The Sass Boilerplate uses the `HTML5 Boilerplate package
+<https://html5boilerplate.com>`_ as its basis. It uses *Gulp* to run the
+compilation, *Bower* as a package manager, and *Node* as a run-time environment.
+
+The advantage of using a dynamic Boilerplate with compiled components is that
+it permits you to build a customised frontend, shorn of items your project does
+not require. Compiled components can also be heavily compressed and optimised,
+while the source files you work on can remain readable and comprehensive.
+
+
+Webpack Boilerplate
+~~~~~~~~~~~~~~~~~~~
+
+The `Webpack Boilerplate
+<https://github.com/divio/djangocms-boilerplate-webpack>`_ implements the
+Bootstrap framework as a fully-compiled frontend set-up. It uses *Gulp* to run
+the compilation, *Webpack* as a package manager, and *Node* as a run-time
+environment.
+
+
+How Boilerplates work
+---------------------
+
+When you create a new project via the Divio Cloud Control Panel, you select a
+Boilerplate at the same time.
+
+The Control Panel examines the Boilerplate to determine what components should
+then be installed.
+
+It does this by checking:
+
+* The ``boilerplate.json`` file (required in all Boilerplates). If templates are
+specified here, they will be applied to the django CMS configuration as
+available templates.
+
+* The Boilerplate's ``Dockerfile``. Sections in the ``Dockerfile`` will be
+  copied to the project's ``Dockerfile`` appropriately; for example::
+
+    # <NODE>
+    ADD build /stack/boilerplate
+
+    ENV NODE_VERSION=6.10.1 \
+        NPM_VERSION=3.10.10
+
+    RUN bash /stack/boilerplate/install.sh
+
+    ENV NODE_PATH=$NVM_DIR/versions/node/v$NODE_VERSION/lib/node_modules \
+        PATH=$NVM_DIR/versions/node/v$NODE_VERSION/bin:$PATH
+    # </NODE>
+
+  will be copied to the::
+
+    # <NODE>
+    # </NODE>
+
+  section.
+
+The Control Panel will then copy all files (other than the two mentioned above)
+and directories in the Boilerplate into the project, unless they are explicitly
+excluded in the the ``boilerplate.json`` file's ``excluded`` list.
+
+The Control Panel will also detect and respond automatically to the presence of
+various other files in the Boilerplate. These files are:
+
+* ``package.json`` - will be used by ``npm`` to install node packages
+* ``bower.json`` - will be used by Bower to install frontend components
```

Comment:
we should definitely strongly recommend to use npm instead

Reply:
not only because it's not being actively maintained, but also because it makes projects bigger, slower and there are sometimes confused people who don't know where it installs and so on

Reply:
also, judging by the code in cp it also actually requires `.bowerrc` configuration file

Reply:
I would add it after that list and say "Optional" but we highly encourage to use NPM